applications/FluidDynamicsApplication/tests/embedded_piston_test.py

```diff
@@ -0,0 +1,224 @@
+import math
+import KratosMultiphysics
+import KratosMultiphysics.MeshMovingApplication
```

Comment:
this import seems to be unused => the solver imports it if the fm-ale-operations are required

Reply:
Removed.